opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -38,16 +36,26 @@
 public class MockTracer implements Tracer {
     private List<MockSpan> finishedSpans = new ArrayList<>();
     private final Propagator propagator;
+    private Scheduler scheduler;
 
     public MockTracer() {
         this(Propagator.PRINTER);
     }
 
+    public MockTracer(Scheduler scheduler) {
+        this(scheduler, Propagator.PRINTER);
+    }
+
+    public MockTracer(Scheduler scheduler, Propagator propagator) {
```

Comment:
nit: could you please define this as the last constructor. (starting from simple to more complex one).